code/src/components/MoviePage.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react'
+
+import { useParams } from 'react-router-dom'
+import { Button } from './Button'
+
+export const MoviePage = () => {
+  const { movieId } = useParams()
+  const [movie, setMovie] = useState([]);
+  const [loading, setLoading] = useState(true)
+  
+  useEffect(() => {
+    const MOVIE_URL = `https://api.themoviedb.org/3/movie/${movieId}?api_key=3bc84b0ae751d27c186bde28023ce8fe&language=en-US`
+    fetch( MOVIE_URL )
+      .then((response) => response.json())
+      .then((json) => setMovie(json))
+      .catch((error) => console.error('Request failed', error))
+  }, [movieId])
+
+  useEffect(() => {
+    if(movie.id) setLoading(false)
+  }, [movie]);
+
+  
+
+  return (
+    <>
```

Comment:
No need for React fragment here since you return only one element - `.movie-page-background`